src/bp/core/services/migration/index.ts

```diff
@@ -119,7 +119,7 @@ export class MigrationService {
 
     await this.database
       .knex('srv_migrations')
-      .insert({ ...entry, details: logs.join('\n'), created_at: this.database.knex.date.now() })
+      .insert({ ...entry, details: logs.join('\n').slice(-255), created_at: this.database.knex.date.now() })
```

Comment:
super quick fix of .slice(-255) because migration has a varchar(255) for details,

needs a migration for this and I dont have time now